nvflare/app_opt/pt/quantization/constant.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
sorry for my earlier comment, let's keep original year for existing files

Reply:
Reverted